tests/unit/test_strategies.py

```
@@ -756,3 +756,187 @@ def test_acceptance_averaging_exact_values(self):
+        )
+        new_step_size = updated_resources["local_sampler"].step_size
+        assert jnp.all(new_step_size > initial_step_size), (
+            "Step size should increase after warmup period"
+        )
+    
+    def test_adapt_step_size_works_with_different_kernels(self):
+        """Test that AdaptStepSize works with MALA, HMC, and GRW."""
+        from flowMC.strategy.adapt_step_size import AdaptStepSize
+        
+        # Test with each kernel type
+        for kernel, target_rate, kernel_name in [
```

Ruff (0.14.13)

Loop control variable `kernel_name` not used within loop body

Rename unused `kernel_name` to `_kernel_name`

(B007)